app/src/main/java/com/example/movieapp/data/MovieRepository.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.movieapp.data
+
+import android.util.Log
+import com.example.movieapp.models.MovieDto
+
+class MovieRepository (
+    private val localDataSource: MovieLocalDataSource,
+    private val remoteDataSource: MovieRemoteDataSource
+) {
+
+    suspend fun loadPopularMovies(page: Int = 0): List<MovieDto> {
```

Comment:
Do not overuse default paramater in functions, are they used?
Here the default is `0` but in remote and local data source its `1`.

ViewModel also keeps the `currPage` as default value of 1

Reply:
Done